code/modules/mob/living/carbon/human/life.dm

```diff
@@ -610,6 +610,19 @@
 			if(prob(nausea))
 				delayed_vomit()
 
+		if(CE_ITCH in chem_effects)
+			var/itching = chem_effects[CE_NOITCH]
+			if(CE_ITCH in chem_effects)
+				itching -= chem_effects[CE_NOITCH]
+			if(itching > 1)
+				if(prob(5))
+					to_chat(src, SPAN_NOTICE(pick("You have an annoying itch.", "You have a slight itch.")))
+			if(itching > 3)
+				if(prob(10))
+					to_chat(src, SPAN_WARNING(pick("The itch is becoming progressively worse.", "You need to scratch that itch!", "The itch isn't going!")))
+					src.emote("scratch")
```

Comment:
Does this actually work? I'm pretty sure it doesn't, and you should make this emote a decl.

Reply:
Also, fyi, you don't need to put `src.` before procs. It looks better without it.

Reply:
Ended up removing the emote here in the end. The flavour messages are good on their own, imo.